pom.xml

```diff
@@ -234,7 +234,8 @@
     <version.com.datastax.cassandra>3.0.0</version.com.datastax.cassandra>
     <!-- keep in sync with modules/system/layers/base/com/fasterxml/jackson/core of the current WildFly/EAP -->
     <version.com.fasterxml.jackson.core>2.5.4</version.com.fasterxml.jackson.core>
-    <version.com.google.guava>16.0.1</version.com.google.guava>
+    <!-- keep in sync with modules/system/layers/base/com/google/guava of the current WildFly/EAP -->
+    <version.com.google.guava>18.0</version.com.google.guava>
```

Comment:
Does this work with the cassandra driver we use? I remember having conflicting versions, but not sure where it was.